tests/testthat/test-bit64.R

```diff
@@ -1,81 +1,76 @@
   expect_true(all((s_act == s_ref)$to_r()))
-  #sereis to R
-  r_act = s_act$to_r()
-  r_ref = bit64::as.integer64(values)
-  expect_identical(
-    r_act, r_ref
-  )
 
-  #lit scalar
-  expect_identical(pl$lit(bit64::as.integer64(5))$to_r(),bit64::as.integer64(5))
-  expect_identical(pl$lit(bit64::as.integer64(NA))$to_r(),bit64::as.integer64(NA))
+  # sereis to R
+  r_act <- s_act$to_r()
+  r_ref <- bit64::as.integer64(values)
+  expect_identical(r_act, r_ref)
 
-  #lit series
-  expect_identical(pl$lit(bit64::as.integer64(c(NA,5)))$to_r(),bit64::as.integer64(c(NA,5)))
+  # lit scalar
+  expect_identical(pl$lit(bit64::as.integer64(5))$to_r(), bit64::as.integer64(5))
+  expect_identical(pl$lit(bit64::as.integer64(NA))$to_r(), bit64::as.integer64(NA))
+
+  # lit series
+  expect_identical(pl$lit(bit64::as.integer64(c(NA, 5)))$to_r(), bit64::as.integer64(c(NA, 5)))
 })
 
 
 test_that("robj_to! from bit64", {
-    testthat::skip_if_not_installed("bit64")
-    expect_identical(
-      unwrap(polars:::test_robj_to_i64(bit64::as.integer64(1))),
-      as.character(bit64::as.integer64(1))
-    )
-
-    expect_identical(
-      unwrap(polars:::test_robj_to_u32(bit64::as.integer64(2^27))),
-      as.character(2^27)
-    )
-
-    expect_identical(
-      unwrap(polars:::test_robj_to_usize(bit64::as.integer64("2305843009213693952"))),
-      as.character(2^61)
-    )
+  skip_if_not_installed("bit64")
+  expect_identical(
+    unwrap(test_robj_to_i64(bit64::as.integer64(1))),
+    as.character(bit64::as.integer64(1))
+  )
 
-    expect_identical(
-      unwrap(polars:::test_robj_to_usize(bit64::as.integer64(2^61))),
-      as.character(2^61)
-    )
+  expect_identical(
+    unwrap(test_robj_to_u32(bit64::as.integer64(2^27))),
+    as.character(2^27)
+  )
 
-    #NO NA
-    expect_grepl_error(
-      unwrap(polars:::test_robj_to_i64(bit64::as.integer64(NA))),
-      "scalar arguments do not support integer64 NA value"
-    )
-    expect_grepl_error(
-      unwrap(polars:::test_robj_to_usize(bit64::as.integer64(NA))),
-      "scalar arguments do not support integer64 NA value"
-    )
+  expect_identical(
+    unwrap(test_robj_to_usize(bit64::as.integer64("2305843009213693952"))),
+    as.character(2^61)
+  )
 
-    #NO OVERFLOW
-    expect_grepl_error(
-      unwrap(polars:::test_robj_to_u32(2^57)),
-      "the value 144115188075855870 cannot exceed u32\\:\\:MAX 4294967295"
-    )
+  expect_identical(
+    unwrap(test_robj_to_usize(bit64::as.integer64(2^61))),
+    as.character(2^61)
+  )
 
-    #NO NEGATIVE
-    expect_grepl_error(
-      unwrap(polars:::test_robj_to_usize(bit64::as.integer64(-1))),
-      "the value -1 cannot be less than zero"
-    )
-    expect_grepl_error(
-      unwrap(polars:::test_robj_to_u32(bit64::as.integer64(-1))),
-      "the value -1 cannot be less than zero"
-    )
+  # NO NA
+  expect_error(
+    unwrap(test_robj_to_i64(bit64::as.integer64(NA))),
+    "scalar arguments do not support integer64 NA value"
+  )
+  expect_error(
```

Comment:
If sytematically replacing the `expect_grepl_error` with `expect_error` one need to address that `expect_error` by it self cannot verify r-polars error messages, as the literally expected text will be included in the call also, and the returned error message can never fail. This silently breaks the usefulness of the test.

Options are: rework unwrap() and the provided context of error message or manually turn on/off the same option expect_grepl_error is doing automatically.

Reply:
I think I misunderstood you before, I didn't realize that:
```r
# NO NA
expect_error(
    unwrap(test_robj_to_i64(bit64::as.integer64(NA))),
    "hello there"
)
```
would not fail. I'll revert the changes made on `expect_grepl_error()`